05-extra.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: page
+title: Version Control with Git
+subtitle: Additional Topics
```

Comment:
Never mind, found it. Still, it seems weird that in the *optional* extras file, suddenly minutes is required, whereas the other git-novice chapters omit it ...